### DISCUSSION TITLE
Include ERC20 decimals in useBalance result

### DISCUSSION THREAD

**Discussion by shrugs** [wevm/wagmi#114] [Ideas]
any time someone is fetching a `useBalance` of a token, they almost certainly also want `decimals` in order to format `value` correctly. wagmi includes symbol, which is excellent, but it would make sense to also include a fetch for `decimals` in this request so users don't need to construct their own erc20 contract instance and fetch it themselves

**Comment by tmm** [comment]
Good idea! https://github.com/tmm/wagmi/pull/122
